### ISSUE TITLE
Unable to find image ‘cdecker/lightningd:master'

### ISSUE THREAD

**Issue by drfalcon** [ElementsProject/lightning#1064]
Installing of lightning with `mkdir -p /scratch/bitcoin/mainnet/clightning` 

While command the next step:

```
docker run --rm --name lightning --network container:bitcoind_mainnet -v /scratch/bitcoin/mainnet/bitcoind:/root/.bitcoin -v /scratch/bitcoin/mainnet/clightning:/root/.lightning --entrypoint /usr/bin/lightningd cdecker/lightningd:master --network=bitcoin --rgb=(mycode) --alias=(mynodename) --ipaddr=(myIP) --log-level=debug
```

```
Unable to find image ‘cdecker/lightningd:master’ locallydocker: Error response from daemon: manifest for cdecker/lightningd:master not found.See ‘docker run — help'
```

**Comment by drfalcon** [comment]
While command the next step: `docker run --rm --name lightning --network container:bitcoind_mainnet -v /scratch/bitcoin/mainnet/bitcoind:/root/.bitcoin -v /scratch/bitcoin/mainnet/clightning:/root/.lightning --entrypoint /usr/bin/lightningd cdecker/lightningd:master --network=bitcoin --rgb=(mycode) --alias=(mynodename) --ipaddr=(myIP) --log-level=debug` it gave me this

**Comment by cdecker** [comment]
I decided to remove the `master` label in favor of having just one label `latest`, if you change the `cdecker/lightningd:master` part to `cdecker/lightningd:latest` it should work again.

**Comment by drfalcon** [comment]
@cdecker Thank you so much, I changed it and yes it works 👍 💯

**Comment by cdecker** [comment]
@drfalcon thanks for the update :+1:

**Comment by yamanmsr** [comment]
it works for me thanks

**Comment by drfalcon** [comment]
Hi Mr. @cdecker , when I want to fund any channel, I get this message: Cannot afford funding transaction

whats the problem, what can I do?

**Comment by drfalcon** [comment]
Yes Sir, I forget to `settxfee` and I set and after connect to new node, It solved. Thanks for replying :)

**Comment by drfalcon** [comment]
Hi Sir again. I want to create a new issue but not sure, so I wrote here again. 
About LN ı have some questions;
**1** How much are the min amount to fund a channel? How can we know this? 
**2** I had created a channel to a node and I sent 0.002 btc. Is it high? Can I get back this fund? 
**3** what is the optimal fee amount? `settxffe` Can I reset the previous channel's fee?

AND THANKS FOR HELPİNG :D
